Submitted by:            Matt Burgess <matthew at linuxfromscratch.org>
Date:                    2010-01-03
Initial Package Version: 2.6.1
Upstream Status:         Submitted
Origin:                  Matt Burgess
Description:             Prevents a test from failing when ed is not available.

diff -Naur patch-2.6.1.orig/tests/crlf-handling patch-2.6.1/tests/crlf-handling
--- patch-2.6.1.orig/tests/crlf-handling	2009-12-30 12:56:30.000000000 +0000
+++ patch-2.6.1/tests/crlf-handling	2010-01-03 11:06:00.340849916 +0000
@@ -89,10 +89,14 @@
 
 # --------------------------------------------------------------
 
-diff -e a b > ab.diff
-cp a c
-check 'patch c < ab.diff' <<EOF
+if ! have_ed ; then
+    echo "The ed utility is not available; skipping ed related tests"
+else
+    diff -e a b > ab.diff
+    cp a c
+    check 'patch c < ab.diff' <<EOF
 EOF
+fi
 
 check 'cat -A c' <<EOF
 1b^M$
